guides/svg/template.md

```diff
@@ -0,0 +1,228 @@
+shape you want.
+
+**Important :** you need to fill it with the `fill` attribute and set a `component-hitbox`class.
+
+```jsx
+<svg class="template" width="250" height="200">
+    <!-- Hitbox -->
+    <rect class="component-hitbox" width="100%" height="100%" fill="white" />
+</svg>
+```
+
+### 3. Add the icon container :
+
+The icon is the small icon that appears on the component, it is usually a `<svg></svg>` with a `component-icon` class.
+
+**Important :** you need to set the `id` attribute with the `{{id}}` syntax.
+
+```jsx
+<svg class="template" width="250" height="200">
+    <rect class="component-hitbox" width="100%" height="100%" fill="white" />
+
+    <!-- Icon container -->
+    <svg id="icon-{{id}}" class="component-icon" width="32" height="32"></svg>
+</svg>
+```
+
+*Note : the icon will be added and the viewBox will be set by the drawer, you don't need to add it manually.*
+
+### 4. Add the metadata you want display :
+In our case we want to display the name and the type of the component, so we add two `<text></text>` with the 
+`component-name` and `component-type` classes.
+
+**Important :** we need to use `nunjucks` syntax to render the name and the type.
+
+```jsx
+<svg class="template" width="250" height="200">
+    <rect class="component-hitbox" width="100%" height="100%" fill="white" />
+  
+    <svg id="icon-{{id}}" class="component-icon" width="32" height="32"></svg>
+
+    <!-- Metadata -->
+    <text class="component-name" y="1em">{{ name }}</text>
+    <text class="component-type" y="2.5em">{{ definition.type }}</text>
+</svg>
+```
+
+### 5. Add the container ***(only for container component)*** :
+To add a container, you need to add a `<svg></svg>` with a `component-container` class, and inside it, you need to add
+a `<rect></rect>` with a `container-background` class.
+
+**Important :** you need to set the `overflow` attribute to `visible` to be able to see the children when you drag them
+outside the container. You also need to fill the `container-background` with the `fill` attribute for drop detection.
+
+```jsx
+<svg class="template" width="250" height="200">
+    <rect class="component-hitbox" width="100%" height="100%" fill="white" />
+  
+    <svg id="icon-{{id}}" class="component-icon" width="32" height="32"></svg>
+
+    <text class="component-name" y="1em">{{ name }}</text>
+    <text class="component-type" y="2.5em">{{ definition.type }}</text>
+
+    <!-- Container -->
+    <svg class="component-container" overflow="visible">
+        <rect class="container-background" width="100%" height="100%" fill="whitesmoke"/>
+    </svg>
+</svg>
+```
+
+### 6. Add the error icon :
+This step is not mandatory, but if you want to display an error icon when the component has an error, you need to add
+a `<g></g>` with a `component-error` class. We used `nunjucks` to have a conditional rendering. For the component has an
+error, we display the error icon. Else, we don't display anything.
```

Comment:
For the component has an error, we display the error icon. Else, we don't display anything. 
-> change to 
When the component has an error, we display the error icon. Otherwise, we don't.

Reply:
Done